docs/11-circleci/03-build.mdx

```diff
@@ -0,0 +1,683 @@
+---
+toc_max_heading_level: 5
+---
+
+# Build
+
+The Unity orb provides the
+[build job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-build) to facilitate the
+process of building your Unity project to multiple platforms.
+
+### Parameters
+
+In this sub-section, you will find information on each parameter the job provides.
+
+#### build-name
+
+- _Default_: `UnityBuild`
```

Comment:
Do you mean to use the repo name as the value for this parameter if it's left empty?

Reply:
I thought we were passing no build name, and Unity would figure it out by itself.

However this is what our logic really looks like in `unity-builder` ([ref](https://github.com/game-ci/unity-builder/blob/c15f955669bd556aae56eb5e5425a263ac058c9c/src/model/input.ts#L112)):

![image](https://user-images.githubusercontent.com/20756439/186034612-83cb5326-36cd-4674-b7b5-75a4f29b83ad.png)

Reply:
Yea, the build name is a required parameter. The execute method I'm using in the orb fails if the parameter comes empty ([ref](https://github.com/game-ci/unity-orb/blob/main/src/dependencies/unity-builder/BuildCommand.cs#L76)):

```c#
static string GetBuildName()
{
    string buildName = GetArgument("customBuildName");
    Console.WriteLine(":: Received customBuildName " + buildName);
    if (buildName == "")
    {
        throw new Exception("customBuildName argument is missing");
    }
    return buildName;
}
```

Reply:
Yea, that's why we're falling back to targetPlatform (which is the only required parameter besides credentials in GitHub).

It would be nice if not everyone had to put a build name if they didn't want to. Target platform could be a good default here. (and if you wanna use the CLI later, you'd not have a breaking change if you make it like that now already)

Reply:
Done ✔️